pages/docs/project management/project-closure.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Project Closure Checklists
+section: Project Management
+order: 1
+---
+
+# Project Closure Checklists
+While closing a project in truemark, here are the list of recommended actions that one should do:
+
+### Project Retrospective and Feedback Collection: Internal Retrospectives and Client, AM & Team Members feedback.
+- Project Retrospectives complete (internal retros).
+- Ask the HR manager to send out a form to be filled up for Client & AM (Account Manager) and team members feedback.
+
+### Comprehensive Project’s Technical Documentation, Outstanding Tasks and Project Closure Report Compilation.
+- Technical documentation in the platform of choice (eg:confluence, GitHub/GitLab).
+- Credentials for hosting and domain stored.
+- Mention what’s still left to do like bugs, requirements etc.
+- Prepare Project Closure Report (*This is handled in separate document*).
+#### Action Items (Developer)
+    1. Developers should create comprehensive technical documentation, including project setup, database structure, API details, deployment procedures, and credentials for production, staging, and development environments, as well as project hosting and domain information, tailored to the specific RBAC or ABAC requirements of each project (This documentation contents could vary depending upon the nature and complexity of projects).
+    2. Developers should review the existing documentation from the relevant personnel and update it to include any changes or additions.
+    3. Decide on the platform for hosting the technical documentation and transfer documentation to the chosen platform: platform includes Confluence, GitHub, or GitLab.
+    4. The decision to choose the platform is taken by the CEO or CTO incase of internal projects and the client incase of client projects.
+#### Action Items (Project Manager)
+    1. The Project Manager should assess and prioritize project bugs and outstanding tasks, assign responsible team members, set deadlines, and communicate progress to stakeholders to ensure that everyone is aware of the remaining work and any potential impact on project closure.
+    2. The Project Manager should prepare a separate document specifically for the project closure report, review and finalize the report.
+
+### Project Closure: Archiving, Access Revocation, Ownership Transfer, and Data Cleanup.
+- JIRA or Trello project archived (optional) and revoke access to people if needed.
```

Comment:
@Kirti2254 optional would be confusing..we should write on what sort of scenarios we can skip archiving